src/views/SelectionInfoView.tsx

```diff
@@ -51,9 +51,35 @@ export default function SelectionInfoView(
         else angleOutput = <p>Unknown angle (missing marker)</p>;
     }
     else angleOutput = <></>; //no output: not enough selected
+    /* Always display force metadata when a force file is loaded */
+    const forcesMetadata = <>
+        {
+            /* Forces, when present, are always at index 0 and 1 */
+            [0,1].map(idx => {
+                const force = forceData?.frames[frame]?.forces[idx];
+                return <p key={"f"+(idx+1)}>
+                    {
+                        force === undefined ?
+                            <></> :
+                            <>
+                                Force {idx+1}<br />
+                                px: {signedValAsString(force.position.x)}<br />
```

Comment:
might be better to have more clear labels here

Reply:
What do you suggest? These are the column labels from the provided data. Width is a concern with each component fitting on a single line; perhaps we could do something like this?:

Force 1
[position]
x: +0.14248
y: +0.051538
z: +0.285417
[components]
x: +8.438947
y: +51.364319
z: +2.782955
[torque]
y: +0.213182

Reply:
yeah that seems reasonable